common/protob/messages-common.proto

```diff
@@ -49,8 +49,13 @@ message Failure {
  * @next ButtonAck
  */
 message ButtonRequest {
-    optional ButtonRequestType code = 1;  // enum identifier of the screen
+    optional ButtonRequestType code = 1;  // enum identifier of the screen (deprecated)
     optional uint32 pages = 2;            // if the screen is paginated, number of pages
+
+    // NB: we skip 3 because it existed briefly: https://github.com/trezor/trezor-firmware/commit/1012ee8497b241e8ca559e386d936fa549bc0357
```

Comment:
that's called `reserved 3;` :)

Reply:
https://github.com/trezor/trezor-firmware/pull/3998/commits/88a99d7b3837c5e7d7074cd0a9fc21b85f5bff40#diff-edf8dee024fcc08dde5dce69057350d825ff7b1c31a12fb68f493571b9291f0dR55